packages/visualizations/src/components/Chart/Chart.svelte

```diff
@@ -111,7 +111,10 @@
                         const percentaged = options?.axis?.assemblage?.percentaged ? '% ' : '';
                         // If the value is a percentage, we need to format the raw value
                         const formattedRawValue =
-                            percentaged && raw && typeof raw === 'number' && `(${format(raw)})`;
+                            percentaged &&
+                            raw !== undefined &&
+                            typeof raw === 'number' &&
+                            `(${format(raw)})`;
```

Comment:
Here, we could remove the explicit `raw !== undefined` check since typeof `raw === 'number'` already ensures that `raw` is defined and is a number. If raw was `undefined`, the type check would return `undefined`, and the expression would evaluate as `false`.

If you want to make sure `raw` is not only a number but also not `NaN`, you can add a check for that:

``` JS
const formattedRawValue = percentage && typeof raw === 'number' && !isNaN(raw) ? `(${raw})` : `(0)`;
```

Reply:
Shoudn't `isNaN(0) === true` ?

Reply:
Yes indeed